test/FakeServer.js

```diff
@@ -95,7 +95,7 @@ FakeConnection.prototype.handshake = function(options) {
   var packetOptions = common.extend({
     scrambleBuff1       : Buffer.from('1020304050607080', 'hex'),
     scrambleBuff2       : Buffer.from('0102030405060708090A0B0C', 'hex'),
-    serverCapabilities1 : 512, // only 1 flag, PROTOCOL_41
+    serverCapabilities1 : 512 | 1 << 11, // only 2 flags, PROTOCOL_41 and SSL
```

Comment:
We don't want to enable this for all the tests; if you have a specific test that needs it, that is why this part of the code will allow it to be overwritten by the given handshake options.

Reply:
excellent - will update